src/NUnitTestAdapter/NUnitTestExecutor.cs

```diff
@@ -61,17 +61,18 @@ public void RunTests(IEnumerable<string> sources, IRunContext runContext, IFrame
                 foreach (var source in sources)
                 {
                     string sourceAssembly = source;
-
                     if (!Path.IsPathRooted(sourceAssembly))
-                    {
                         sourceAssembly = Path.Combine(Environment.CurrentDirectory, sourceAssembly);
-                    }
-                    using (currentRunner = new AssemblyRunner(TestLog, sourceAssembly, tfsfilter,this))
+
+                    try // TODO: Do we need try?
```

Comment:
I don't see the point of this unless we are catching. `currentRunner` will not get used if we throw, so there is no need to set it to null. I don't even see the point in setting to null anymore since we are looping and resetting it anyway. Do we however want to catch and send an error message so that additional test assemblies get processed?